src/useq/_mda_sequence.py

```diff
@@ -183,6 +215,7 @@ def _check_order(
         z_plan: Optional[AnyZPlan] = None,
         stage_positions: Sequence[Position] = (),
         channels: Sequence[Channel] = (),
+        tile_plan: Optional[AnyTilePlan] = None,
```

Comment:
this doesn't appear to be used anywhere in this function?

Reply:
That's true!